docs/components/Header.js

```diff
@@ -36,26 +72,83 @@ function Header() {
               <Flex alignItems="center" gap={2}>
                 <GestaltLogo height={40} width={40} />
                 {/* slight tweak to vertically center to logo */}
-                <Box dangerouslySetInlineStyle={{ __style: { marginBottom: '1px' } }}>
+                <Box
+                  display="none"
+                  mdDisplay="block"
+                  dangerouslySetInlineStyle={{ __style: { marginBottom: '1px' } }}
+                >
                   Gestalt Design System
                 </Box>
               </Flex>
             </Box>
           </Link>
         </Text>
-        <Text size="sm" weight="bold">
-          v{GestaltPackageJson.version}
-        </Text>
+        <Box paddingX={2}>
```

Comment:
@ayeshakmaz 

![Uploading Kapture 2022-05-25 at 13.30.14.gif…]()

Reply:
![Kapture 2022-05-25 at 13 30 14](https://user-images.githubusercontent.com/10593890/170327318-c797dafa-8c6c-4200-91a0-9d3969610e8a.gif)

Reply:
Yeah, the Tooltips don't play nicely when header is wrapped in ScrollBoundaryContainer